src/Stack/Types/BuildPlan.hs

```diff
@@ -269,6 +270,26 @@ instance subdirs ~ Subdirs => FromJSON (WithJSONWarnings (PackageLocation subdir
             , archiveHash = msha'
             }
 
+        github = withObjectWarnings "PLArchive" $ \o -> do
```

Comment:
It should probably say `PLArchive:github` or something like that instead, this is just used for error messages, and it would be useful to know what part of the code failed the parsing.

Reply:
Done